src/AutoMerge.Tests/AutoMerge.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Could you please add with condition on $(VisualStudioVersion) it like in the line below so we don't break it for VS2017.